ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py

```diff
@@ -6,22 +6,25 @@
 ```
```

Comment:
probably need a shebang first line of the file
```#!/usr/bin/env python3```
without it i was unable to run directly
```
❯ get_missed_upgrade_jobs.py 
get_missed_upgrade_jobs.py': [Errno 2] No such file or directory
get_missed_upgrade_jobs.py: line 14: $'\nThis script is used to find out missed e2e profiles in upgrade config file.\nWhen we adding new upgrade config files, we can compare e2e config file with upgrade config file, then find out the jobs have not automated in upgrade.\n\nHere is the example to run this script:\n\n': command not found
get_missed_upgrade_jobs.py: line 22: syntax error near unexpected token `('
get_missed_upgrade_jobs.py: line 22: `def get_jobs(yaml_file, mode='y'):'
```

Reply:
Because I am think a use always run this script on his local machine, so the shebang is not required, anyway I will add it.